be/src/exprs/vectorized/string_functions.cpp

```diff
@@ -889,31 +891,37 @@ void fast_repeat(uint8_t* dst, const uint8_t* src, size_t src_size, int32_t repe
 
 static inline ColumnPtr repeat_const_not_null(const Columns& columns, const BinaryColumn* src) {
     auto times = ColumnHelper::get_const_value<TYPE_INT>(columns[1]);
-    auto& src_bytes = src->get_bytes();
+
     auto& src_offsets = src->get_offset();
     const auto num_rows = src->size();
-    const auto bytes_size = src_bytes.size();
 
-    auto result = BinaryColumn::create();
-    auto& dst_bytes = result->get_bytes();
-    auto& dst_offsets = result->get_offset();
+    NullableBinaryColumnBuilder builder;
+    auto& dst_nulls = builder.get_null_data();
+    auto& dst_offsets = builder.data_column()->get_offset();
+    auto& dst_bytes = builder.data_column()->get_bytes();
+
+    dst_nulls.resize(num_rows);
+    bool has_null = false;
+
+    constexpr auto LIMIT = std::numeric_limits<size_t>::max();
```

Comment:
Could use OLAP_STRING_MAX_LENGTH?

Reply:
all right

Reply:
max repeat times was int32, slice.size won't exceed 2^32. so we don't have to check overflow in `s.size * n`